jsEx/Form.js

```diff
@@ -0,0 +1,106 @@
+function validate() {
+  let username = document.getElementById("username").value;
+  let password = document.getElementById("password").value;
+  let confirmPassword = document.getElementById("confirm-password").value;
+
+  let isValidUser = isValidUsername(username);
+  let isValidPass = isValidPassword(password);
+  let isValidConfirmPass = isValidConfirmPassword(confirmPassword, password);
+
+  if (isValidUser) {
+    hideErrorMessag("invalidUsername");
+  }
+  if (isValidPass) {
+    hideErrorMessag("invalidPassword");
+  }
+  if (isValidConfirmPass) {
+    hideErrorMessag("invalidConfirmPassword");
+  }
+
+  return (isValidUser && isValidPass && isValidConfirmPass) ? request() : false;
```

Comment:
Not a strong feedback, no action required - the code is generally ok: It is usually a good idea to strive for more readable code than for shorter code, e.g. here we have in the return statement itself - boolean operations, ternary operator based on the result of the boolean operations result and a function being executed that is also returning a result.